device-types/HPE/ProLiant-DL180-Gen6.yaml

```diff
@@ -0,0 +1,36 @@
+---
+manufacturer: HPE
+model: ProLiant DL180 Gen6
+slug: hp_dl180_g6
+u_height: 2.0
+is_full_depth: true
+part_number: 590638-371
+comments: '[HP ProLiant DL180 G6 QuickSpecs](https://www.hpe.com/psnow/doc/c04284378)'
+console-ports:
```

Comment:
Do you have any info on the `weight` and `airflow`? If so, please add that

Reply:
Added a `weight` based on the maximum set(all hard drives, power supplies, and processors installed)
```
weight:
 - value: 16.80
   unit: kg
```